test-runner-frontend/src/Views/Dashboard.test.js

```diff
@@ -0,0 +1,51 @@
+import Dashboard from './Dashboard';
+import { render, fireEvent, waitFor, screen, getByRole, queryHelpers, getByTestId } from '@testing-library/react';
+import userEvent from '@testing-library/user-event';
+
+describe('Dashboard', () => {
+  /*
+  test('Check buttons rendering and functionality', async () => {
+    const { debug, getByRole } = render(<Dashboard/>);
+    const CompareButton = getByRole('button', { name: /Compare/i });
+    expect(CompareButton).toBeDefined();
+    await userEvent.click(CompareButton);
+    const CancelButton = getByRole('button', { name: /Cancel/i });
+    expect(CancelButton).toBeDefined();
+    await userEvent.click(CancelButton);
+  });
+  */
```

Comment:
please remove any comments that aren't needed anymore

Reply:
Done